File: src/bubbleSort.js
export default function bubbleSort( initialSet ){
  if( initialSet == null ){
    return 'Cannot sort null'
  }
  if( initialSet.constructor !== Array ){
    return 'Please use Arrays'
  }

  let end = initialSet.length

  while(end > 1){
    for( let i = 1; i < end; i++){
      if( initialSet[i-1] > initialSet[i]){
        let temp = initialSet[i]
        initialSet[i] = initialSet[i-1]
        initialSet[i-1] = temp
      }
    }
    end--
  }

  return initialSet
}

File: python/bubbleSort.py
def bubbleSort( initialSet ):
    end = len(initialSet)
    while end >1 :
        for i in range (1, end):
            if initialSet[i-1] > initialSet[i]:
                temp = initialSet[i]
                initialSet[i] = initialSet[i-1]
                initialSet[i-1] = temp
        end -= 1
    print initialSet
    return initialSet

bubbleSort([6,2,4,7,1,9,10,5,3,8])
